lib/handleRequest.js

```diff
@@ -46,7 +47,11 @@ function handleRequest (req, res, params, context) {
     return
   }
 
-  reply.code(405).send(new Error('Method Not Allowed: ' + method))
+  if (supportedMethods.indexOf(method) > -1) {
+    reply.code(404).send(new Error('Not Found'))
```

Comment:
This code is unreachable because all of the "supported" methods are handled in the `if` statements above.

Reply:
Mmm, you are right.
We must decide how the framework should behave here.
When a request with an unsupported method comes in, what should we do?
Return a 405 or a 404?